docs/core-concepts/libraries/libraries-with-cli.md

```diff
@@ -4,94 +4,83 @@ By [Phillip Carter](https://github.com/cartermp)
 2. You can use a lower version of the .NET Platform Standard to target earlier .NET platforms and versions. The cost here is not having access to some of the latest APIs. Example targets:
 
     - .NET Framework 4.5.2, 4.5.1, or 4.5
     - Windows Phone
     - Windows Phone Silverlight
     - Universal Windows Platform
-    - DNX Core
     - Xamarin Platforms
-    - Mono 
+    - Mono
 
-    [Refer to the platform mapping table](https://github.com/dotnet/corefx/blob/master/Documentation/project-docs/standard-platform.md#mapping-the-net-platform-standard-to-platforms) to choose the `dotnetXX` moniker you need to target.
+    [Refer to the platform mapping table](https://github.com/dotnet/corefx/blob/master/Documentation/architecture/net-platform-standard.md#mapping-the-net-platform-standard-to-platforms) to choose the `netstandardXX` moniker you need to target.
     
-    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `dotnet54`.
+    For example, if you wanted to have compatibility with .NET Core and .NET Framework 4.6, you would pick `netstandard1.3`.
 
-    ```javascript
+    ```json
     {
+        "dependencies":{
+            "NETStandard.Library":"1.5.0-rc2-24027"
+        },
         "frameworks":{
-            "dotnet54":{}
+            "netstandard1.3":{}
         }
     }
     ```
     
-    For example, you need to target `dotnet51` to support Windows Phone Silverlight 8. The .NET Platform Standard versions in a backward-compatible way. That means that `dotnet51` libraries run on `dotnet52` platforms and later (e.g. .NET Framework 4.6 and .NET Core). When targeting `dotnet51`, you get the benefit of more platform reach than `dotnet52` but have access to fewer APIs. Each .NET Platform Standard version adds APIs and drops platforms. You should select the Standard version that has the right mix of APIs and platform support for your needs.
+    The .NET Platform Standard versions in a backward-compatible way. That means that `netstandard1.0` libraries run on `netstandard1.1` platforms and higher. When targeting `netstandard1.0`, you get the benefit of more platform reach than `netstandard1.1` but have access to fewer APIs. Each .NET Platform Standard version adds APIs and drops platforms. You should select the Standard version that has the right mix of APIs and platform support for your needs.
     
-3. If you want to target the .NET Framework versions 4.0 or below, or you wish to use an API available in .NET Framework but not in .NET Core (e.g. `System.Drawing`), you'll need to read the next sections.
+3. If you want to target the .NET Framework versions 4.0 or below, or you wish to use an API available in the .NET Framework but not in .NET Core (for example, `System.Drawing`), you'll need to read the following sections and learn how to multitarget.
```

Comment:
you'll need -> you need
or 
"you'll need to read the following sections and learn how to multitarget" -> "read the following sections to learn how to multitarget"